lib/src/modern_player_options.dart

```diff
@@ -314,6 +314,19 @@ class ModernPlayerCustomActionButton {
       {required this.icon, this.onPressed, this.onDoubleTap, this.onLongPress});
 }
 
+typedef DefaultSelector = bool Function(int index, String label);
+
+class ModernPlayerDefaultSelectionOptions {
+  DefaultSelector? defaultSubtitleSelector;
+  DefaultSelector? defaultAudioSelector;
+  DefaultSelector? defaultQualitySelector;
```

Comment:
These could be ordered arrays to support fallback logic.

Example: I prefer "English" for subtitle, but it should fallback to "None" as opposed to falling back to the first one, which is "Italian"